lib/puppet/application/lookup.rb

```diff
@@ -57,7 +57,6 @@ class Puppet::Application::Lookup < Puppet::Application
     options[:node] = arg
   end
 
-  # not yet supported
   option('--facts FACT_FILE') do |arg|
     if %w{.yaml .yml .json}.include?(arg.match(/\.[^.]*$/)[0])
```

Comment:
There's an else statement here (that GitHub is hiding) which will raise and error if the wrong type of file is passed in. Does that seem correct?